msprime/trees.py

```diff
@@ -1770,7 +1778,7 @@ def sequence_length(self):
         genomic scale over which tree coordinates are defined. Given a
         tree sequence with a sequence length :math:`L`, the constituent
         trees will be defined over the half-closed interval
-        :math:`(0, L]`. Each tree then covers some subset of this
+        :math:`[0, L)`. Each tree then covers some subset of this
```

Comment:
elsewhere everything is closed-on-the-left

Reply:
My fault: I can never remember which bracket means which.